bridge/device/cloud/manager.go

```diff
@@ -384,12 +387,12 @@ func (c *Manager) dial(ctx context.Context) error {
 		options.WithMaxMessageSize(c.maxMessageSize),
 		options.WithBlockwise(false, blockwise.SZX1024, time.Second*4),
 		options.WithErrors(func(err error) {
-			log.Printf("error: %v\n", err)
+			c.logger.Errorf("error: %v\n", err)
```

Comment:
`c.logger.Errorf("error: %v\n", err)` -> `c.logger.Errorf("cloud connection error: %w", err)`

-> remove `\n` in all logs
-> and format error via `fmt.Errorf` in stdlogger -> (https://github.com/plgd-dev/device/pull/434/files#diff-94b349782ae6f41cb5f84505c022a9177cbc09347c2c769bf403272e2f33c0f2R142)